curator/patterns/hidden-post-terms-and-date.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Title: Tags and date
+ * Slug: curator/post-terms-and-date
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"10px"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group">
+<!-- wp:post-date {"isLink":true} /-->
+<!-- wp:post-terms {"term":"post_tag"} /-->
```

Comment:
Neither the category nor tag is rendering for me right now, I think it should be the following: 

```suggestion
<!-- wp:post-terms {"term":"category"} /-->
```